lib/ansible/modules/files/template.py

```diff
@@ -13,156 +17,152 @@
+    - This can be a relative or an absolute path.
+    required: yes
   dest:
     description:
-      - Location to render the template to on the remote machine.
-    required: true
+    - Location to render the template to on the remote machine.
+    required: yes
   backup:
     description:
-      - Create a backup file including the timestamp information so you can get
-        the original file back if you somehow clobbered it incorrectly.
+    - Influence whether a backup should be created.
+    - When set to C(yes), create a backup file including the timestamp information
+      so you can get the original file back if you somehow clobbered it incorrectly.
     type: bool
-    default: 'no'
+    default: no
   newline_sequence:
     description:
-      - Specify the newline sequence to use for templating files.
+    - Specify the newline sequence to use for templating files.
     choices: [ '\n', '\r', '\r\n' ]
     default: '\n'
     version_added: '2.4'
   block_start_string:
     description:
-      - The string marking the beginning of a block.
+    - The string marking the beginning of a block.
     default: '{%'
     version_added: '2.4'
   block_end_string:
     description:
-      - The string marking the end of a block.
+    - The string marking the end of a block.
     default: '%}'
     version_added: '2.4'
   variable_start_string:
     description:
-      - The string marking the beginning of a print statement.
+    - The string marking the beginning of a print statement.
     default: '{{'
     version_added: '2.4'
   variable_end_string:
     description:
-      - The string marking the end of a print statement.
+    - The string marking the end of a print statement.
     default: '}}'
     version_added: '2.4'
   trim_blocks:
     description:
-      - If this is set to True the first newline after a block is removed (block, not variable tag!).
+    - Influence when newlines should be removed from blocks.
+    - When set to C(yes) the first newline after a block is removed (block, not variable tag!).
     type: bool
-    default: 'yes'
+    default: yes
     version_added: '2.4'
   lstrip_blocks:
     description:
-      - If this is set to True leading spaces and tabs are stripped from the start of a line to a block.
-        Setting this option to True requires Jinja2 version >=2.7.
+    - Influence when leading spaces and tabs should be stripped.
+    - When set to C(yes) leading spaces and tabs are stripped from the start of a line to a block.
+    - This functionality requires Jinja v2.7 or newer.
     type: bool
-    default: 'no'
+    default: no
     version_added: '2.6'
   force:
     description:
-      - the default is C(yes), which will replace the remote file when contents
-        are different than the source.  If C(no), the file will only be transferred
-        if the destination does not exist.
+    - Influence when the file is being transferred if the destination already exists.
```

Comment:
I'd use "determine if the file is transferred" - "influence" sounds like the setting could be overridden by another paramreter.

Reply:
Great, thanks !